docs/querying/datasource.md

```diff
@@ -302,12 +302,15 @@ use join datasources.
 SQL joins take the form:
 
 ```
-<o1> [ INNER | LEFT [OUTER] ] JOIN <o2> ON <condition>
+<o1> [ INNER | LEFT [OUTER] | CROSS ] JOIN <o2> ON <condition>
 ```
 
-The condition must involve only equalities, but functions are okay, and there can be multiple equalities ANDed together.
-Conditions like `t1.x = t2.x`, or `LOWER(t1.x) = t2.x`, or `t1.x = t2.x AND t1.y = t2.y` can all be handled. Conditions
-like `t1.x <> t2.x` cannot currently be handled.
+JOIN conditions have the following requirements:
+- You can only use equality statements in JOIN conditions. For example: `t1.x = t2.x`
```

Comment:
"You can only use" --> "Druid supports only"

To make it clear that, although you might have used other conditions in other SQL tools, Druid supports only equality.

Also "equality statements" -> "the equality operator"

In SQL, a "statement" is the entire SELECT or INSERT construct. Here we're talking about an "operator", specifically a boolean binary operator.

Reply:
```suggestion
- Druid only supports the equality operator, `=`, in JOIN conditions. For example: `t1.x = t2.x`
```